components/ui-shell/index.ts

```diff
@@ -18,6 +18,9 @@ import styles from './index.scss';
 import { Variant, NAV_ITEMS, SOCIAL_LINKS } from './settings.js';
 import type { NavItem, TopLevelNavItem } from './settings.js';
 
+const BASE_URL =
+  window.origin === '${BASE_URL}/' ? 'https://qiskit.org/' : window.origin;
```

Comment:
Just working locally because `'${BASE_URL}/' ? 'https://qiskit.org/'` is always false.

```suggestion
const BASE_URL =
  window.origin === 'https://qiskit.org'
    ? 'https://learn.qiskit.org'
    : window.origin;
```